lib/ansible/modules/network/fortios/fortios_system_link_monitor.py

```diff
@@ -0,0 +1,486 @@
+    - Run as a local_action in your playbook
+requirements:
+    - fortiosapi>=0.9.8
+options:
+    host:
+        description:
+            - FortiOS or FortiGate IP address.
+        type: str
+        required: false
+    username:
+        description:
+            - FortiOS or FortiGate username.
+        type: str
+        required: false
+    password:
+        description:
+            - FortiOS or FortiGate password.
+        type: str
+        default: ""
+    vdom:
+        description:
+            - Virtual domain, among those defined previously. A vdom is a
+              virtual instance of the FortiGate that can be configured and
+              used as a different unit.
+        type: str
+        default: root
+    https:
+        description:
+            - Indicates if the requests towards FortiGate must use HTTPS protocol.
+        type: bool
+        default: true
+    ssl_verify:
+        description:
+            - Ensures FortiGate certificate must be verified by a proper CA.
+        type: bool
+        default: true
+    state:
+        description:
+            - Indicates whether to create or remove the object.
+        type: str
+        choices:
+            - present
+            - absent
+    system_link_monitor:
+        description:
+            - Configure Link Health Monitor.
+        default: null
+        type: dict
+        suboptions:
+            addr_mode:
+                description:
+                    - Address mode (IPv4 or IPv6).
+                type: str
+                choices:
+                    - ipv4
+                    - ipv6
+            failtime:
+                description:
+                    - Number of retry attempts before the server is considered down (1 - 10, default = 5)
+                type: int
+            gateway_ip:
+                description:
+                    - Gateway IP address used to probe the server.
+                type: str
+            gateway_ip6:
+                description:
+                    - Gateway IPv6 address used to probe the server.
+                type: str
+            ha_priority:
+                description:
+                    - HA election priority (1 - 50).
+                type: int
+            http_agent:
+                description:
+                    - String in the http-agent field in the HTTP header.
+                type: str
+            http_get:
+                description:
+                    - If you are monitoring an HTML server you can send an HTTP-GET request with a custom string. Use this option to define the string.
+                type: str
+            http_match:
+                description:
+                    - String that you expect to see in the HTTP-GET requests of the traffic to be monitored.
+                type: str
+            interval:
+                description:
+                    - Detection interval (1 - 3600 sec, default = 5).
```

Comment:
Default mentioned in description but not specified in argspec.

Reply:
As discussed, @mamunozgonzalez will be fixing this as a part of a consolidated PR.